bin/savepdf

```diff
@@ -20,6 +20,8 @@ program.arguments('<input>')
   .option('-o, --output <output_file>',
           `specify output file path (default output.pdf)`,
           'output.pdf')
+  .option('-r, --root <root_directory>',
```

Comment:
`--root` option will affect both headless mode and preview mode, so this option should have appear above the `--preview` option line. Please note also this line 👉  https://github.com/pentapod/viola-savepdf/pull/8/files#diff-03085efc0ce458fb546b974a26ac7eb7R19)